lambdas/process-async-cri-credential/src/main/java/uk/gov/di/ipv/core/processasynccricredential/auditing/AuditRestrictedVc.java

```diff
@@ -1,29 +1,28 @@
 package uk.gov.di.ipv.core.processasynccricredential.auditing;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
-import com.fasterxml.jackson.databind.JsonNode;
+import com.nimbusds.jose.shaded.json.JSONArray;
```

Comment:
Why are we using a different JSON library here - feels a little odd to mix? Can we use the jackson version, or just a standard Java list?

Reply:
It's because nimbus is the lib we use for handling JWTs, and it uses these JSON types when digging into the objects to pull out various elements.
Previously we were doing this, and then converting them to a string, then converting them back to a Jackson data type  to include in the AuditRestrictedVc object.
This change means we can skip the serialize/deserialize step and just use the nimbus object we already have. Jackson can serialize it without issue when the while audit event is serialized.
We could use a Java list, but then I think we'd end up with `List<JSONObject>` which is just using another nimbus object. Unless we want to reintroduce the serialize/deserialize.

Reply:
Ok fair enough - I don't love having both libraries mixed, but mapping to the other type is probably a waste of time.

It also means our audit event structure is tightly coupled to the VC structure, but that's probably not such a massive issue (if anything it makes sense for them to match).